shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm

```diff
@@ -17,10 +17,29 @@ @interface FlutterPlatformPluginTest : XCTestCase
 
 @interface FlutterPlatformPlugin ()
 - (BOOL)isLiveTextInputAvailable;
+- (void)showLookUpView:(NSString*)term;
 @end
 
 @implementation FlutterPlatformPluginTest
 
+- (void)testLookUpCallInitiated {
+  FlutterEngine* engine = [[[FlutterEngine alloc] initWithName:@"test" project:nil] autorelease];
+  std::unique_ptr<fml::WeakPtrFactory<FlutterEngine>> _weakFactory =
+      std::make_unique<fml::WeakPtrFactory<FlutterEngine>>(engine);
+  XCTestExpectation* invokeExpectation = [self expectationWithDescription:@"isLookUpInvoked"];
```

Comment:
The string should be a sentence that gets print out when test fails. It's not a variable so don't use camel case.